docs/vf-setup.md

```diff
@@ -0,0 +1,152 @@
+# Setting up Virtual Functions
```

Comment:
Why is this shown as new file? This was already added in #240 
Am I missing something?